examples/get-started/flutter-for/ios_devs/lib/navigation.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(
+    // create an instance of our app
+    // and passes it to the runApp function
+    const App(),
+  );
+}
+
+// #docregion Routes
+// defines the route name as a constant
```

Comment:
I think this is a useful comment, but format to Effective Dart guidelines - Capitals, periods, etc. 

https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences